flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/schema/MySqlSchema.java

```diff
@@ -70,36 +72,100 @@ public TableChange getTableSchema(JdbcConnection jdbc, TableId tableId) {
 
     private TableChange readTableSchema(JdbcConnection jdbc, TableId tableId) {
         final Map<TableId, TableChange> tableChangeMap = new HashMap<>();
+        String showCreateTable = readSchemaByShowCreateTable(jdbc, tableId, tableChangeMap);
+        if (!tableChangeMap.containsKey(tableId)) {
+            // fallback to desc table
+            String descTable = readSchemaByDescTable(jdbc, tableId, tableChangeMap);
```

Comment:
return a command string is wired with the function name,   we can define `private static final String SHOW_CREATE_TABLE = "SHOW CREATE TABLE "` and  `private static final String DESC_TABLE = "DESC "` and then construct the `showCreateTable` and `descTable` sql.

Reply:
thank, have fixed based on your suggestion .